src/Tribe/Utils/DST.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+
+class Tribe__Events__Utils__DST {
+
+	/**
+	 * Whether the time is in DST or not.
+	 *
+	 * @var bool
+	 */
+	protected $_in_dst;
+
+	/**
+	 * @var string A UNIX timestamp
+	 */
+	protected $time;
+
+	/**
+	 * @var bool
+	 */
+	protected $is_dst_cache;
+
+	/**
+	 * Tribe__Events__Utils__DST constructor.
+	 *
+	 * @param int|string $time    Either a UNIX timestamp or an english format date.
+	 * @param null|bool  $_in_dst An injectable DST status, meant for tests.
+	 */
+	public function __construct( $time, $_in_dst = null ) {
+		if ( ! is_numeric( $time ) ) {
+			$time = strtotime( $time );
+		}
+
+		$this->time    = $time;
+		$this->_in_dst = $_in_dst;
+	}
+
+	/**
+	 * Whether the current time is in DST or not.
+	 *
+	 * @return bool
+	 */
+	public function is_in_dst() {
+		//wwid
```

Comment:
orgh. It means "What was I doing?"; I use it to mark what I was working on before stopping, removing